src/esbmc/bmc.cpp

```diff
@@ -288,6 +289,30 @@ void bmct::report_trace(
   }
 }
 
+void bmct::report_multi_property_trace(
+  smt_convt::resultt &res,
+  const std::string &msg)
+{
+  assert(options.get_bool_option("base-case"));
```

Comment:
Why does it work only for the base case?

Reply:
There is nothing preventing other modes. I just didn't implement the messages for them.

As of now, It didn't make sense for me. Maybe we could somehow reuse the unwinding assertions that were proven, that's beyond this PR though.

Reply:
Could you please add a TODO here?